docs/sources/panels-visualizations/visualizations/canvas/index.md

```diff
@@ -42,7 +42,19 @@ The rectangle element enables you to add a basic rectangle to the canvas. Rectan
 
 The icon element enables you to add a supported icon to the canvas. Icons can have their color set based on thresholds / value mappings.
 
-## Canvas Editing
+### Server
+
+The server element enables you to easily represent a single server, a stack of servers, a database, or a terminal. Server elements support status color, bulb color, and a bulb blink rate all configurable by fixed or field values.
+
+{{< figure src="/media/docs/grafana/canvas-server-element-9-4-0.png" max-width="750px" caption="Canvas server element" >}}
+
+## Connections
+
+When building a canvas panel, you can connect elements together to create more complex visualizations. Connections are created by dragging from the connection anchor of one element to the connection anchor of another element. You can also create connections to the background of the canvas panel. Connection anchors are displayed when you hover over an element and inline editing is turned on. To remove a connection, simply click on the connection directly and then press the "Delete" key.
```

Comment:
Delete key or backspace, right?

Reply:
yes - do you think it is worth mentioning both?

Reply:
Do we have a tooltip when a connection is highlighted that explicitly provides these instructions? If not, I think it is worth mentioning both in the docs for completeness.